iceoryx_posh/source/gateway/toml_gateway_config_parser.cpp

```diff
@@ -36,7 +36,6 @@ iox::config::TomlGatewayConfigParser::parse(const roudi::ConfigFilePathString_t&
         return iox::cxx::success<GatewayConfig>(config);
     }
 
-    /// @todo Replace with C++17 std::filesystem::exists()
```

Comment:
We should create a C++17 issue!

Reply:
No this is over engineering. Our file class solves the problem as well. The code the user would have to write with the new approach would even increase - but I admit it would be a little bit more readable.

Reply:
You understood me wrong, I didn't me a C++17 todo issue which is linked in lots of different places in the code. But instead an issue for C++17 where we collect all cool features that we're longing for to use :man_technologist:

Reply:
I created one, see: https://github.com/eclipse-iceoryx/iceoryx/issues/1718